test/features/deploy_quarkus_service.feature

```diff
@@ -51,6 +51,31 @@ Feature: Deploy quarkus service
       | native  | minutes |
       | enabled | 20      |
 
+#####
+
+  Scenario Outline: Deploy process-optaplanner-quarkus service without persistence and native <native>
```

Comment:
are you planning springboot in another PR ?

Reply:
I am not - for the very same reason I stated in the previous comment.

> If we are going to test all examples, I think we should create a new testing subset of features that should be executed on demand. Or should we test them all in full testing ?
> @sutaakar @Sgitario @MarianMacik what do you think ?

We test examples that incorporate processes and rules, but the OptaPlanner is completely missing here right now.

After we talked about testing all examples and you guys didn't recommend it, I decided to add only a single test - to be sure that OptaPlanner is tested in combination with Quarkus and OptaPlanner. I don't plan to add anything else.